pkg/monitor/datapath_trace.go

```diff
@@ -126,9 +140,9 @@ func (n *TraceNotify) traceSummary() string {
 
 // DumpInfo prints a summary of the trace messages.
 func (n *TraceNotify) DumpInfo(data []byte) {
-	fmt.Printf("%s flow %#x identity %d->%d state %s ifindex %s: %s\n",
-		n.traceSummary(), n.Hash, n.SrcLabel, n.DstLabel,
-		connState(n.Reason), ifname(int(n.Ifindex)), GetConnectionSummary(data[TraceNotifyLen:]))
+	fmt.Printf("%s %sflow %#x identity %d->%d state %s ifindex %s: %s\n",
```

Comment:
wouldn't this be `%s-flow` instead of `%sflow`?

Reply:
Yeah that will print nicer. Will push update.